Source/Tools/FEXLoader/FEXLoader.cpp

```diff
@@ -194,15 +194,49 @@ void RootFSRedirect(fextl::string* Filename, const fextl::string& RootFS) {
   }
 }
 
+FEX::Config::PortableInformation GetIsPortable() {
+  constexpr FEX::Config::PortableInformation BadResult {false, {}};
+  const char* PortableConfig = getenv("FEX_PORTABLE");
+  if (!PortableConfig) {
+    return BadResult;
+  }
+
+  uint32_t Value {};
+  std::string_view PortableView {PortableConfig};
+
+  if (std::from_chars(PortableView.data(), PortableView.data() + PortableView.size(), Value).ec != std::errc {} || Value == 0) {
+    return BadResult;
+  }
+
+  char SelfPath[PATH_MAX];
+  auto Result = readlink("/proc/self/exe", SelfPath, PATH_MAX);
+  if (Result == -1) {
+    return BadResult;
+  }
+
+  std::string_view SelfPathView {SelfPath, std::min<size_t>(PATH_MAX, Result)};
+
+  ///< Extract the absolute path from the FEXInterpret path
+  return {true, SelfPathView.substr(0, SelfPathView.find_last_of("/") + 1)};
+}
+
 bool RanAsInterpreter(const char* Program) {
   return ExecutedWithFD || FEXLOADER_AS_INTERPRETER;
 }
 
+static bool InterpreterInstalled {};
+
 bool IsInterpreterInstalled() {
-  // The interpreter is installed if both the binfmt_misc handlers are available
-  // Or if we were originally executed with FD. Which means the interpreter is installed
+  return InterpreterInstalled;
+}
 
-  return ExecutedWithFD || (access("/proc/sys/fs/binfmt_misc/FEX-x86", F_OK) == 0 && access("/proc/sys/fs/binfmt_misc/FEX-x86_64", F_OK) == 0);
+void SetInterpreterInstalled(bool Portable) {
```

Comment:
This might as well take `PortableInformation` directly, which avoids boolean blindness and doesn't cost us anything:
```suggestion
void SetInterpreterInstalled(const PortableInformation& Portable) {
```

Reply:
Switched over